Source/OxyPlot/Axes/Axis.cs

```diff
@@ -543,6 +543,11 @@ public double Scale
         public bool UseSuperExponentialFormat { get; set; }
 
         /// <summary>
+        /// Gets or sets the natural dimensions of the axis, as of most recent rendering.
+        /// </summary>
+        public OxySize NaturalSize { get; protected set; }
```

Comment:
Isn't ActualSize used to indicate the size that was chosen for rendering?  NaturalSize indicates the "natural" size of the element if involved no clipping / size override.   Correct me if I am wrong, if size is overriden by the user then Actual variables will reflect the override.